airbyte-integrations/connectors/source-postgres/build.gradle

```diff
@@ -9,7 +9,7 @@ plugins {
 airbyteJavaConnector {
     cdkVersionRequired = '0.1.9'
     features = ['db-sources']
-    useLocalCdk = false
+    useLocalCdk = true
```

Comment:
Don't forget to publish the CDK, import the new version and set this to `false` before merging.

Reply:
Ah yes! Just wanted to do that to make sure tests were passing!

Reply:
Am publishing it, making the switch after it's been published and works locally with the newly published version